### PR TITLE
74 simulate execution of external call

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**

### CODE REVIEW

File: evm_loader/cli/src/account_storage.rs [hunk 1]
```diff
@@ -103,6 +116,7 @@ pub struct EmulatorAccountStorage<'a> {
     caller_id: H160,
     block_number: u64,
     block_timestamp: i64,
+    // pub external_call_fail: bool,
```
Comment (s-medvedev) [inline evm_loader/cli/src/account_storage.rs:119]: Please, remove unused commented code.

File: evm_loader/test_cli_emulate.py [hunk 2]
```diff
@@ -245,171 +353,293 @@ def test_successful_cli_emulate(self):
...
+        src_data = result['result']['meta']['innerInstructions'][-1]['instructions'][-1]['data']
+        self.assertEqual(base58.b58decode(src_data)[0], 6)  # 6 means OnReturn
+        self.assertLess(base58.b58decode(src_data)[1], 0xd0)  # less 0xd0 - success
+
+        self.assertEqual(self.spl_token.balance(self.token_acc1), balance1 + mint_amount - 3*transfer_amount/2)
+        self.assertEqual(self.spl_token.balance(self.token_acc2), balance2 + 2*transfer_amount/2)
+        self.assertEqual(self.spl_token.balance(self.token_acc3), balance3 + 1*transfer_amount/2)
+
+    @unittest.skip("a.i.")
+    def test_emulate_external_call_by_rpc(self):
```
Comment (s-medvedev) [inline evm_loader/test_cli_emulate.py:611]: What do the symbols `a.i.` mean?
What does this test check?

Comment (vasiliy-zaznobin) [reply evm_loader/test_cli_emulate.py]: a.i. = temporarily
Remove the test.

File: evm_loader/cli/src/account_storage.rs (hunk 1 above)
Comment (vasiliy-zaznobin) [reply evm_loader/cli/src/account_storage.rs]: Fixed.

File: evm_loader/test_cli_emulate.py [hunk 3]
```diff
@@ -139,12 +185,12 @@ def compare_tmpl_and_emulate_result(self, tmpl_json, emulate_result):
         self.assertEqual(tmpl_json['result'], emulate_result['result'])
 
         self.compare_accounts(tmpl_json, emulate_result)
+        self.compare_solana_accounts(tmpl_json, emulate_result)
 
-    def compare_tmpl_and_emulate_result_2(self, tmpl_json, emulate_result):
-        self.compare_tmpl_and_emulate_result(tmpl_json, emulate_result)
-        self.compare_account_metas(tmpl_json, emulate_result)
-
+    # @unittest.skip("a.i.")
```
[the remark this answers is not in the document]
Comment (vasiliy-zaznobin) [reply evm_loader/test_cli_emulate.py:190]: Fixed.

File: Dockerfile [hunk 4]
```diff
@@ -12,6 +12,7 @@ FROM builder AS evm-loader-builder
 COPY ./evm_loader/ /opt/evm_loader/
 WORKDIR /opt/evm_loader/program
 RUN cargo clippy
+#RUN cargo build-bpf
```
[the remark this answers is not in the document]
Comment (vasiliy-zaznobin) [reply Dockerfile]: Fixed.

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**